vivisect/parsers/elf.py

```diff
@@ -474,7 +504,7 @@ def loadElfIntoWorkspace(vw, elf, filename=None, baseaddr=None):
     # Now that the program and section headers have been parsed, if this is a
     # PowerPC 32-bit ELF we need to check for VLE flags
     if arch in ppc_arch_names:
-        maps = vw.getMeta('PpcVleMaps')
+        maps = vw.getMeta('PpcMemoryMaps')
```

Comment:
I will change this meta and all the supporting functions to be `PpcVlePages` to match the new name of the new configuration option.